phase1/openstack/configure-vm.sh

```diff
@@ -5,6 +5,17 @@ set -o pipefail
 set -o nounset
 set -o xtrace
 
+KUBEADM_TOKEN="${k8s_kubeadm_token}"
+KUBEADM_VERSION="${k8s_kubeadm_version}"
+KUBEADM_KUBERNETES_VERSION="${k8s_kubeadm_kubernetes_version}"
+KUBEADM_KUBELET_VERSION="${k8s_kubeadm_kubelet_version}"
+KUBEADM_ENABLE_CLOUD_PROVIDER="${k8s_kubeadm_enable_cloud_provider}"
```

Comment:
This cloud-provider is not currently working. The config file also needs to be added

kubelet.service error
```
error: failed to run Kubelet: could not init cloud provider "openstack": no OpenStack cloud provider config
```

Reply:
cloud-provider for openstack was not available even before this pr. So lets fix it in follow-up PR. @ss22ever interested in picking this up?

Reply:
@shashidharatd thanks, yes I would like to.